src/components/Header.js

```diff
@@ -105,7 +110,7 @@ export default class Header extends Component {
                   <li className='nav-item dropdown'>
                     <a
                       className='nav-link'
-                      href='#pablo'
+                      href='javascript:void(0)'
```

Comment:
what does it do?

Reply:
https://stackoverflow.com/questions/22940761/best-way-to-create-an-a-link-with-empty-href 
Just decided to replace link to not existing pablo page, but this link should lead nowhere) and it's a way how to use empty href